cmd/tailscale/cli/serve_v2.go

```diff
@@ -39,15 +39,22 @@ type commandInfo struct {
 }
 
 var serveHelpCommon = strings.TrimSpace(`
-<target> can be a port number (e.g., 3000), a partial URL (e.g., localhost:3000), or a
-full URL including a path (e.g., http://localhost:3000/foo, https+insecure://localhost:3000/foo).
+<target> can be a file, directory, text, or most commonly the location to a service running on the
+local machine. The location to the location service can be expressed as a port number (e.g., 3000),
+a partial URL (e.g., localhost:3000), or a full URL including a path (e.g., http://localhost:3000/foo).
 
 EXAMPLES
-  - Mount a local web server at 127.0.0.1:3000 in the foreground:
-    $ tailscale %s localhost:3000
+  - Expose an HTTP serve running at 127.0.0.1:3000 in the foreground:
```

Comment:
@samlinville thoughts on the change of wording away from using "mount"?

Reply:
Fine with it — what's your suggestion for an alternative?

Reply:
It's in the PR, mostly using the wording from the CLI flags.

Reply:
SGTM